sympy/geometry/entity.py

```diff
@@ -100,6 +100,7 @@ def __contains__(self, other):
         raise NotImplementedError()
 
     def __getnewargs__(self):
+        """Returns a Tuple that will be passed to __new__ on unpickling."""
```

Comment:
Actually, it returns a tuple; Tuple is a SymPy object while tuple is a Python object.